gwpy/timeseries/tests/test_timeseries.py

```diff
@@ -55,10 +55,7 @@
 SKIP_LALFRAME = utils.skip_missing_dependency('lalframe')
 SKIP_PYCBC_PSD = utils.skip_missing_dependency('pycbc.psd')
 
-if scipy_version < '1.2.0':
-    SCIPY_METHODS = ('welch', 'bartlett')
-else:
-    SCIPY_METHODS = ('welch', 'bartlett', 'median')
+SCIPY_METHODS = ('welch', 'bartlett', 'median')
```

Comment:
I think this variable can be removed entirely now, since its not really variable any more, any just replace its usage (decorator to `test_psd`) with the tuple.

Reply:
Makes sense to me, thanks @duncanmmacleod, this is done in the latest commit